cmd/picoshare/main.go

```diff
@@ -82,3 +92,16 @@ func ensureDirExists(dir string) {
 func isLitestreamEnabled() bool {
 	return os.Getenv("LITESTREAM_BUCKET") != ""
 }
+
+func setupSignalHandler() (stopCh <-chan struct{}) {
```

Comment:
Can we delete the name `stopCh`? The rest of the code doesn't use named returns, and we don't use `stopCh` in this function.